File: php/testcode/benchmark_test_00026.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00026(BenchmarkRequest $req): BenchmarkResponse {
    $input = $req->param('tpl');
    ob_start();
    eval('?>' . $input . '<?php ');
    $html = ob_get_clean();
    return BenchmarkResponse::ok($html);
}

File: rust/testcode/benchmark_test_00026.rs
const ADMIN_PASSWORD: &str = "Admin@123!";

pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let password = req.param("password");
    if password == ADMIN_PASSWORD {
        super::shared::BenchmarkResponse::ok("Login successful")
    } else {
        super::shared::BenchmarkResponse::forbidden("Invalid password")
    }
}
